6starter.tex

```diff
@@ -210,38 +210,38 @@
 
  \hline
 %  \pagebreak
- 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (aka spawning biomass, aka B). The calculated values are reported to the SD report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
+ 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (a.k.a., \gls{sb}, a.k.a. B). The calculated values are reported to the \gls{sd} report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
    & & \\
    & & \\
    & 0 = skip; & \Tstrut \\
-   & 1 = X*B0; & Relative to virgin spawning biomass. \\
-   & 2 = X*B\textsubscript{MSY}; & Relative to spawning biomass that achieves MSY. \\
-   & 3 = X*B\textsubscript{styr}; and & Relative to model start year spawning biomass. \\
-   & 4 = X*B\textsubscript{endyr}. & Relative to spawning biomass in the model end year. \\
-   & 5 = X*Dynamic B0 & Relative to the calculated dynamic B0. \\
+   & 1 = $X*B0$; & Relative to virgin spawning biomass. \\
+   & 2 = $X*B_{MSY}$; & Relative to spawning biomass that achieves \gls{msy}. \\
+   & 3 = $X*B_{styr}$; and & Relative to model start year spawning biomass. \\
+   & 4 = $X*B_{endyr}$. & Relative to spawning biomass in the model end year. \\
+   & 5 = $X*Dynamic B0$ & Relative to the calculated dynamic B0. \\
    & use tens and hundreds digits to invoke multi-year trailing average & \\
    & append 0.1 to invoke log(ratio) & \Bstrut\\
   
  \hline
- 1.0 & Fraction (X) for depletion denominator & Value for use in the calculation of the ratio for SB\textsubscript{y}/(X*SB0). \Tstrut\Bstrut\\
+ 1.0 & Fraction (X) for depletion denominator & Value for use in the calculation of the ratio for $SB_{y}/(X*SB0)$. \Tstrut\Bstrut\\
```

Comment:
I think that "SB0" should be "SB_{0}"

Reply:
Okay, I'll apply this to B0 and R0 too.

Reply:
This would be a question for @Rick-Methot-NOAA imo. But, I see them as time series where the integer references which point in time the Biomass or Recruitment is coming from. Given that $ln(R_{0})$ is a parameter and not a derived quantity other people might have different ideas about how it should be referenced.